src/plugin/ui/auth-menu.ts

```
@@ -1,153 +1,663 @@
+  | { type: 'settings' }
+  | { type: 'search' }
   | { type: 'cancel' };
 
-export type AccountAction = 'back' | 'delete' | 'refresh' | 'toggle' | 'verify' | 'cancel';
+export type AccountAction = 'back' | 'delete' | 'refresh' | 'toggle' | 'set-current' | 'verify' | 'cancel';
 
-function formatRelativeTime(timestamp: number | undefined): string {
+export function sanitizeTerminalText(value: string | undefined): string | undefined {
+  if (!value) return undefined;
+  return value
+    .replace(/\x1b\[[0-?]*[ -/]*[@-~]/g, '')
```

Biome (2.4.4)

[error] Unexpected control character in a regular expression.

(lint/suspicious/noControlCharactersInRegex)